### PR TITLE
extract content test conf properly

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6317
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6428

## Description
* Extract only packs which exist in content
* Notify in slack when the packs are not synced (there are packs in content-test-conf which are not in the content repo).
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Content nightly
https://code.pan.run/xsoar/content/-/pipelines/5110242

### CODE REVIEW

**Comment by daryakoval** [comment]
Please run nightly and put a link here.

**Comment by daryakoval** [comment]
Update me when it is ready for review.

**Review by daryakoval — COMMENTED** [review]
Please add my comment as well, since now it is not working in the way I want.

**Comment by ilaner** [comment]
@daryakoval 
link to build to prove it works:
https://code.pan.run/xsoar/content/-/jobs/25113570

**Review by daryakoval — APPROVED** [review]

File: Tests/scripts/gitlab_slack_notifier.py [hunk 1]
```diff
@@ -199,6 +199,14 @@ def construct_slack_msg(triggering_workflow, pipeline_url, pipeline_failed_jobs)
         content_fields += test_playbooks_results(ARTIFACTS_FOLDER_MPV2, title="XSIAM")
         content_fields += test_playbooks_results(ARTIFACTS_FOLDER_XPANSE, title="XPANSE")
         coverage_slack_msg = construct_coverage_slack_msg()
+        missing_packs = get_artifact_data(ARTIFACTS_FOLDER_XSOAR, 'missing_content_packs_test_conf.txt')
+        if missing_packs:
+            missing_packs = missing_packs.split('\n')
+            content_fields.append({
+                "title": f"{title} - Notice - Missing packs - ({len(missing_packs)})",
+                "value": f"The following packs exist in content-test-conf, but not in content: {', '.join(missing_packs)}",
+                "short": False
```
Comment (daryakoval) [inline Tests/scripts/gitlab_slack_notifier.py:208]: Is it working?

**Pull request merged**